pegasus/sites.v3/hourofcode.com/public/learn/index.haml

```diff
@@ -134,6 +134,7 @@ social:
     var img = $('<img>');
     img.attr('src', studentImage);
     img.attr('style', 'opacity: 0');
+    img.attr('alt', 'Hour of Code events happening all around the globe');
```

Comment:
Have we confirmed the new strings with marketing/product?

Reply:
Looking at this image in context, I wonder if this alt text could also be an empty string? This image is decorative and the associated text provides the necessary context to understand this section.

Reply:
Good call. I'll do that.